lib/bitstyles_phoenix/alpine3/dropdown.ex

```diff
@@ -132,7 +132,7 @@ defmodule BitstylesPhoenix.Alpine3.Dropdown do
   def ui_js_dropdown(assigns) do
     {_button, button_extra} = assigns_from_single_slot(assigns, :button)
 
-    {_menu, menu_extra} = assigns_from_single_slot(assigns, :menu, with: :menu_extra)
+    {_menu, menu_extra} = assigns_from_single_slot(assigns, :menu)
```

Comment:
Passing this `with` option did not do anything as `assigns_from_single_slot` only accepts `exclude` and `optional` as options.